app/helpers/pages_helper.rb

```diff
@@ -33,6 +39,15 @@ def plugin_title(plugin)
     "#{plugin.name}#{detail}"
   end
 
+  def plugin_icon(plugin)
+    registered = {
+      action: 'hand-rock-o',
+      thermometer: 'neuter'
+    }
+    name = plugin.name.underscore.to_sym
+    registered.has_key?(name) ? registered[name] : 'cubes'
```

Comment:
I'm a little torn because I like how concise this is but I find it a bit cryptic. the ternary operator is pretty explicit what it does.